### PR TITLE
initial ui for search result

### PR DESCRIPTION
**Pull request by Devraj91**

### CODE REVIEW

File: ml-search-web/src/app/app.component.css [hunk 1]
```diff
@@ -0,0 +1,22 @@
+.header {
+    background-color: black;
+    box-shadow: 0 0 0 0 rgba(0, 0, 0, 0), 0 0 0 rgba(0, 0, 0, 0);
```
Comment (pankajupadhyay29) [inline ml-search-web/src/app/app.component.css]: need to check this box shadow, this is a black shadow on a black box without any variation on any axis

File: ml-search-web/src/app/app.component.html [hunk 2]
```diff
@@ -1,30 +1,8 @@
...
-    <input type="text">
-    <button (click)="onSearch()">Search</button>
-
-    <div *ngIf="loading$ | async"> Loading...</div>
-    <div *ngFor="let search of searchResults$ | async" class="">
-        <a hrf="#"> {{search.url}}</a>
-        <div>{{search.text}}</div>
-    </div>
-
+<!--
```
Comment (pankajupadhyay29) [inline ml-search-web/src/app/app.component.html]: remove this comment

File: ml-search-web/src/app/search-box/search-box.component.css [hunk 3]
```diff
@@ -0,0 +1,20 @@
+.box {
+    padding-top: 40px;
```
Comment (pankajupadhyay29) [inline ml-search-web/src/app/search-box/search-box.component.css]: why this padding top?

File: ml-search-web/src/app/search-box/search-box.component.css [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    width: 300px;
+}
+
+.btn {
+    height: 40px;
+    margin-left: 5px;
+    width: 60px;
+}
+
+.search {
```
Comment (pankajupadhyay29) [inline ml-search-web/src/app/search-box/search-box.component.css]: create a generic class .inline-block in style.css

File: ml-search-web/src/app/search-box/search-box.component.css [hunk 5]
```diff
@@ -0,0 +1,20 @@
+.box {
+    padding-top: 40px;
+    text-align: center;
+}
+
+.txtwidth {
```
Comment (pankajupadhyay29) [inline ml-search-web/src/app/search-box/search-box.component.css]: improve naming here, it is not just width

File: ml-search-web/src/app/search-container/search-container.component.ts [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+  ngOnInit(): void {
+
+  }
+
+  onSearch(text) {
+    const query: Query = {searchTerm: text, pageNumber: 1};
+    this.store.dispatch(new StartSearchAction(query));
+  }
+
+  onPageChange(searchTerm, pageNumber) {
```
Comment (pankajupadhyay29) [inline ml-search-web/src/app/search-container/search-container.component.ts]: need to introduce pager to use this

File: ml-search-web/src/app/store/actions.ts [hunk 7]
```diff
@@ -16,4 +16,9 @@ export class CompleteSearchAction implements Action {
   constructor(public searchData: SearchData) {}
 }
 
+export class ChangePage implements Action {
```
Comment (pankajupadhyay29) [inline ml-search-web/src/app/store/actions.ts]: do we really need this? On change page we will be searching again with new pageNumber.

File: ml-search-web/src/app/search-container/search-container.component.ts (hunk 6 above)
Comment (shagunsinghal2207) [reply ml-search-web/src/app/search-container/search-container.component.ts]: I am working on this.

File: ml-search-web/src/app/app.component.css (hunk 1 above)
Comment (Devraj91) [reply ml-search-web/src/app/app.component.css]: removed unnecessary styles.

File: ml-search-web/src/app/app.component.html (hunk 2 above)
Comment (Devraj91) [reply ml-search-web/src/app/app.component.html]: removed

File: ml-search-web/src/app/search-box/search-box.component.css (hunk 3 above)
Comment (Devraj91) [reply ml-search-web/src/app/search-box/search-box.component.css]: removed

File: ml-search-web/src/app/store/actions.ts (hunk 7 above)
Comment (Devraj91) [reply ml-search-web/src/app/store/actions.ts]: ChangePage removed as we dont require this

**Pull request merged**